<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmProducto</class>
 <widget class="QDialog" name="frmProducto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1206</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Demo 23: Consulta Desconectada del Producto</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>251</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Registros encontrados</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblTotal">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>100</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="bntExportarExcel">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>90</y>
     <width>150</width>
     <height>46</height>
    </rect>
   </property>
   <property name="text">
    <string>Exportar Excel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSalir">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>90</y>
     <width>150</width>
     <height>46</height>
    </rect>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </widget>
  <widget class="QListWidget" name="lvwProducto">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>991</width>
     <height>431</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblNombre">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>301</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Ingresa Nombre del Producto</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtNombre">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>30</y>
     <width>631</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
